docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -47,55 +44,55 @@ mo --input_model tf_nasnet_large.onnx --layout "nhwc->nchw"
+For example, run the Model Optimizer for the PaddlePaddle UNet model and apply mean-scale normalization to the input data.
 
 ```sh
 mo --input_model unet.pdmodel --mean_values [123,117,104] --scale 255
 ```
 
-## When to Reverse Input Channels <a name="when_to_reverse_input_channels"></a>
+## Reversing Input Channels <a name="when_to_reverse_input_channels"></a>
 Sometimes input images for your application can be of the RGB (BGR) format and the model is trained on images of the BGR (RGB) format,
 the opposite color channel order. In this case, it is important to preprocess the input images by reverting the color channels before inference.
-To embed this preprocessing step into IR, Model Optimizer provides the `--reverse_input_channels` command-line parameter to shuffle the color channels.
+To embed this preprocessing step into IR, Model Optimizer provides the *`--reverse_input_channels`* command-line parameter to shuffle the color channels.
 
-The `--reverse_input_channels` parameter applies to an input of the model in two cases.
+The *`--reverse_input_channels`* parameter applies to an input of the model in two cases.
  * Only one dimension in the input shape has a size equal to 3.
- * One dimension has an undefined size and is marked as `C` channel using `layout` parameters.
+ * One dimension has an undefined size and is marked as *`C`* channel using *`layout`* parameters.
 
-Using the `--reverse_input_channels` parameter, Model Optimizer embeds the corresponding preprocessing block for reverting
-the input data along channel dimension and optimizes this block so that the preprocessing takes negligible time for inference.
+Using the *`--reverse_input_channels`* parameter, Model Optimizer embeds the corresponding preprocessing block for reverting
+the input data along channel dimension and optimizes this block so that the preprocessing takes a little time for inference.
 
-For example, launch the Model Optimizer for the TensorFlow* AlexNet model and embed `reverse_input_channel` preprocessing block into IR.
+For example, launch the Model Optimizer for the TensorFlow AlexNet model and embed *`reverse_input_channel`* preprocessing block into IR.
 
 ```sh
 mo --input_model alexnet.pb --reverse_input_channels
 ```
 
 > **NOTE**: If both mean and scale values are specified, the mean is subtracted first and then the scale is applied regardless of the order of options
-in the command line. Input values are *divided* by the scale value(s). If also `--reverse_input_channels` option is used, the `reverse_input_channels`
-will be applied first, then `mean` and after that `scale`. The data flow in the model looks as follows:
-`Parameter -> ReverseInputChannels -> Mean apply-> Scale apply -> the original body of the model`.
+in the command line. Input values are *divided* by the scale value(s). If also *`--reverse_input_channels`* option is used, the *`reverse_input_channels`*
```

Comment:
Here the formatting of Note breaks in the output. You may want to check it in your editor.